pdns/tcpiohandler.hh

```diff
@@ -9,7 +9,8 @@
 #include "misc.hh"
 #include "noinitvector.hh"
 
-enum class IOState : uint8_t { Done, NeedRead, NeedWrite };
+/* Async is only returned for TLS connections, if OpenSSL's async mode has been enabled */
+enum class IOState : uint8_t { Done, NeedRead, NeedWrite, Async };
```

Comment:
How would you like me to fix that, adding a new case to all switches, throwing an exception if the handler ever returns `IOState::Async`, since it will never happen unless someone writes the code to load an engine and enable the asynchronous mode?

Reply:
An exception for a "can't happen" situation is fine.